redskyctl/internal/commands/export/export.go

```diff
@@ -422,17 +402,19 @@ func (o *Options) generateExperiment() error {
 	}
 
 	// Load up all application resources
-	res, err := apppkg.LoadResources(o.application, onDiskFS)
-	if err != nil {
-		return err
-	}
-
-	resYaml, err := res.AsYaml()
```

Comment:
We can just use a KYAML pipeline and skip the extra round trip into a Kustomize `ResMap`.

Reply:
I think kio pipelines have taken over your mind :)

Reply:
100%